templates/kotlin/src/main/kotlin/io/appwrite/Client.kt.twig

```diff
@@ -286,8 +286,17 @@ class Client @JvmOverloads constructor(
                         .charStream()
                         .buffered()
                         .use(BufferedReader::readText)
+                        
                     val error = if (response.headers["content-type"]?.contains("application/json") == true) {
-                        body.fromJson()
+                        val map = gson.fromJson<Map<String, Any>>(
+                            body,
+                            object : TypeToken<Map<String, Any>>(){}.type
+                        )
+                        {{ spec.title | caseUcfirst }}Exception(
+                            map["message"] as? String ?: "", 
```

Comment:
What does `as?` do?

Reply:
This actually is a change in Kotlin and should be reverted.